src/auth/auth.controller.ts

```diff
@@ -21,4 +24,18 @@ export class AuthController {
     return user;
   }
 
+  @Post('/signout')
+  @ApiOperation({description: 'Sign out a User'})
+  async SignOutUser(@Body('refreshToken') refreshToken: string) {
+    return await this.authService.signout(refreshToken);
+  }
+  
+  @Serialize(UserDto)
```

Comment:
What is this for?

Reply:
@TijanAyo you didnt answer this question

Reply:
I think @SunmadeBabafemi worked on it to process the object before it returns or something of that manner

I haven't really gotten deep in serialisation... 

I saw it in the dummy and I felt it would be important just to exclude

Reply:
@serialize is an interceptor that intercepts the user objects and specifies what parameters to return from the user when the endpoint is hit...